arisculala/backend-coding-test
# Implement pagination for GET: /rides api

* added limit, offset, column, sort to make the endpoint flexible

* install lodash for query parameter validation and setting of default value

1. Clone the remote repository backend-coding-test in your local machine (using the release tag pagination)
        $ git clone https://github.com/arisculala/backend-coding-test.git --branch pagination

2. Go inside the cloned backend-coding-test project
        $ cd backend-coding-test

3. Ensure node (>8.6 and <= 10) and npm are installed

4. Run
        $ npm install

5. Run
		$ npm run start

6. Open in browser
	http://localhost:8010/api/docs/

	- (You should be able to see the additional input query parameters in GET: /rides - The query parameters will be use for pagination purpose)
	- Under 'Schemas' of the swagger UI documentation, you will be able to see the definition and usage of the additional query parameters (limit, offset, column and sort)
